static/app/views/settings/organizationIntegrations/reinstallAlert.tsx

```diff
@@ -0,0 +1,29 @@
+import {ReactNode} from 'react';
+
+import {Alert} from 'sentry/components/alert';
+import {t} from 'sentry/locale';
+import {Integration} from 'sentry/types';
+import {getIntegrationStatus} from 'sentry/utils/integrationUtil';
+
+type Props = React.ComponentPropsWithoutRef<typeof Alert> & {
+  integrations: Integration[] | null;
+  message?: ReactNode;
```

Comment:
are you planning on changing the message in other places? Might not make sense to have it as a prop yet